src/components/GuaranteedAnalysisForm.tsx

```diff
@@ -0,0 +1,83 @@
+import { FormComponentProps, LabelData, UNITS } from "@/types/types";
+import { Box, Typography } from "@mui/material";
+import { useEffect } from "react";
+import { FormProvider, useForm, useWatch } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import VerifiedBilingualTable from "./VerifiedBilingualTable";
+import { VerifiedCheckbox, VerifiedInput } from "./VerifiedFieldComponents";
+
+function GuaranteedAnalysisForm({
+  labelData,
+  setLabelData,
+}: FormComponentProps) {
+  const { t } = useTranslation("labelDataValidationPage");
+  const methods = useForm<LabelData>({
+    defaultValues: labelData,
+  });
+
+  const { control } = methods;
+
```

Comment:
Can you directly get control in line 14?
```JavaScript
const { control } = useForm<LabelData>({
    defaultValues: labelData,
  });
  ```

Reply:
Because methods is used later, I instead use `methods.control`.